build/make.go

```diff
@@ -88,7 +88,7 @@ func createPluginDistro(forAllPlatforms bool) {
 	} else {
 		createDistro()
 	}
-	log.Printf("Distributables created in directory => %s \n", deploy)
+	fmt.Printf("Distributables created in directory => %s \n", deploy)
```

Comment:
Can we use logger package instead of fmt. I think its best if such messages come on the log files too.

Reply:
[build/make.go](https://github.com/getgauge/html-report/blob/structured-logs/build/make.go)  is only used to build aur install `html-plugin` by developer and does not have any role in the final `html-plugin`.

If we use structure logging in this file there is not one to interpret those message.